spec/initializer_spec.rb

```diff
@@ -0,0 +1,53 @@
+require_relative '../lib/initializer.rb'
+require_relative '../lib/board/board.rb'
+require_relative '../lib/board/board_checker.rb'
+require_relative '../lib/players/ai.rb'
+require_relative '../lib/players/human.rb'
+require_relative '../lib/players/player.rb'
+require_relative '../lib/ui/user_interface.rb'
+
+describe 'initializer' do
+  before(:each) do
+    @board = Board.new(3, '-')
+    @checker = BoardChecker.new(3, '-')
+    @script = Script.new
+    @user_interface = UserInterface.new(@board, @script)
+    @ai_logic = AI.new(@checker, 'O', 'X')
+    @human_logic1 = Human.new(@board, @user_interface, 'X')
+    @human_logic2 = Human.new(@board, @user_interface, 'O')
+  end
```

Comment:
These instance variables _work_ fine for the tests, but the rspec convention is to use `let` statements, they're lazy and aren't evaluated until they're utilized in the example.